Projects/Features/SearchFeature/Sources/Reactors/AfterSearchReactor.swift

```diff
@@ -0,0 +1,95 @@
+import Foundation
+import LogManager
+import ReactorKit
+import RxSwift
+import SongsDomainInterface
+
+public final class AfterSearchReactor: Reactor {
+    var disposeBag: DisposeBag = DisposeBag()
+    var fetchSearchSongUseCase: FetchSearchSongUseCase
```

Comment:
```suggestion
    private let fetchSearchSongUseCase: FetchSearchSongUseCase
```
바뀌지도 않고 외부에도 접근하는건 안조흐인 private let 으로 하는거 어떤가요